### PR TITLE
Add FAQ section on reproducing results

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*
#1921 

*Description of changes:*
- Add FAQ section on reproducing results

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/tabular_prediction/tabular-faq.md [hunk 1]
```diff
@@ -139,6 +139,17 @@ task.fit(..., excluded_model_types=['KNN','RF','XT'])
 
 To learn how to add your own custom models to the set that AutoGluon trains, tunes, and ensembles, review :ref:`sec_tabularcustommodel`.
 
+### How can I reproduce the results of model XYZ without using AutoGluon?
+
+Exactly reproducing the results of AutoGluon models without using AutoGluon is extremely challenging.
```
Comment (gradientsky) [inline docs/tutorials/tabular_prediction/tabular-faq.md:144]: `extremely challenging` -> `can be challenging`.

File: docs/tutorials/tabular_prediction/tabular-faq.md [hunk 2]
```diff
@@ -139,6 +139,17 @@ task.fit(..., excluded_model_types=['KNN','RF','XT'])
...
+(Ex: Trying to replicate the results of AutoGluon's LightGBM without depending on AutoGluon)
+AutoGluon uses a variety of techniques not present in the model frameworks it builds upon such as data preprocessing,
+edge case handling, custom early stopping, and custom architectures. These are all used to enhance the performance
+and simplify usage beyond what exists in the original model framework (such as LightGBM, XGBoost, FastAI, etc.).
+This is a core benefit of AutoML frameworks and is inherently complicated to replicate
+(you would have to reimplement much of what AutoGluon does behind the scenes).
```
Comment (gradientsky) [inline docs/tutorials/tabular_prediction/tabular-faq.md:150]: > This is a core benefit of AutoML frameworks and is inherently complicated to replicate
> (you would have to reimplement much of what AutoGluon does behind the scenes).

This sentence is probably not needed.

**Review by yinweisu — APPROVED** [review]
LGTM!

File: docs/tutorials/tabular_prediction/tabular-faq.md [hunk 3]
```diff
@@ -139,6 +139,17 @@ task.fit(..., excluded_model_types=['KNN','RF','XT'])
...
+If you still wish to try to replicate results, you must dive into the source code for a given model
+without guidance from the AutoGluon devs.
```
Comment (gradientsky) [inline docs/tutorials/tabular_prediction/tabular-faq.md:152]: If you still wish to try to replicate results, the logic can be derived from the source code.

Comment (Innixma) [reply docs/tutorials/tabular_prediction/tabular-faq.md]: Updated

File: docs/tutorials/tabular_prediction/tabular-faq.md (hunk 2 above)
Comment (Innixma) [reply docs/tutorials/tabular_prediction/tabular-faq.md:150]: Keeping as I do want to provide reasoning to the user and as a warning to what they are getting themselves into.

File: docs/tutorials/tabular_prediction/tabular-faq.md (hunk 1 above)
Comment (Innixma) [reply docs/tutorials/tabular_prediction/tabular-faq.md]: Updated

**Pull request merged**